send-coin/html/js/index.js

```diff
@@ -0,0 +1,52 @@
+const baseAccount = {
+  address: "A12NSDKGBRyddRhraQGHCgY7XCxUpYjsnkKGqvDoUJHATREcBpco",
+  secretKey: "S1waNKtVExLZZFmFdoYmAaxzFWrTU17GmYMPKejsDiXzA5hfx3Z",
+  publicKey: "P12KZymkagdS1bZbREw2D4dzdNYQK7ENcPg2Q8tYM6AbKPMRAJ2J",
+};
+
+let client = null;
+let massa = window.massa;
+
+massa.ClientFactory.createDefaultClient(
+  massa.DefaultProviderUrls.TESTNET,
+  false,
+  baseAccount
+).then((c) => {
+  client = c;
+  console.log(client);
+});
+
+function strEncodeUTF16(str) {
```

Comment:
Is this being used anywhere ?

Reply:
For that we also have stuff in massa-web3: `import { bytesToStr, strToBytes } from '@massalabs/massa-web3/utils/serializers';`

Reply:
Ah yes you are right that are not used I just forget them over there. Necessary changes has been done.